src/vs/workbench/services/actions/common/menusExtensionPoint.ts

```diff
@@ -321,6 +321,12 @@ const apiMenus: IAPIMenu[] = [
 		id: MenuId.InteractiveCellTitle,
 		description: localize('interactive.cell.title', "The contributed interactive cell title menu"),
 	},
+	{
+		key: 'issue/reporter',
+		id: MenuId.IssueReporter,
+		description: localize('issue.reporter', "The contributed issue reporter menu"),
+		// proposed: 'contribIssueReporter'
```

Comment:
works when commented out, does not when this is in. likely related to https://github.com/microsoft/vscode/pull/205176/files#r1488848619

Reply:
Maybe @jrieken should comment here